ui/src/pages/Dashboard/DashboardPage.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState } from 'react';
+import TabLayout from '@splunk/react-ui/TabLayout';
+import ErrorBoundary from '../../components/ErrorBoundary/ErrorBoundary';
+import { OverviewDashboard } from './Overview';
+import { DataIngestionDashboard } from './DataIngestion';
+import { ErrorDashboard } from './Error';
+import { CustomDashboard } from './Custom';
+import './dashboardStyle.css';
+
+function getBuildDirPath() {
```

Comment:
we have a similar script in `ui/src/util/script.ts`. Is it different?

Reply:
yea, at first we thought it would be separated from main code, so thats why i moved it here, fixed as import from that file